### PR TITLE
Upgrade ocw-data-parser to version 0.29.2

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
N/A

#### What's this PR do?
Upgrade ocw-data-parser to version 0.29.2 that includes the conversion of srt files to vtt

#### How should this be manually tested?
nothing should break

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -18,23 +18,23 @@ base36==0.1.1
...
     # via celery
+boto==2.39.0
+    # via
+    #   -r requirements.in
+    #   smart-open
 boto3==1.14.37
     # via
     #   ocw-data-parser
     #   pyathena
     #   smart-open
```
Comment (pdpinch) [inline requirements.txt:37]: Do we need boto _and_ boto3?

File: requirements.txt [hunk 2]
```diff
@@ -104,12 +114,12 @@ django-filter==2.2.0
...
 django-guardian==1.4.9
     # via -r requirements.in
-django-hijack-admin==2.1.10
-    # via -r requirements.in
 django-hijack==2.1.10
     # via
     #   -r requirements.in
     #   django-hijack-admin
+django-hijack-admin==2.1.10
+    # via -r requirements.in
```
Comment (pdpinch) [inline requirements.txt:122]: Why do we have django-hijack in ocw-data-parser?

**Review by noisecapella — APPROVED** [review]

**Review by noisecapella — CHANGES_REQUESTED** [review]
Actually this needs an additional change. You need to add `create_vtt_files=True` to `OCWParser` in `course_catalog/api` to activate it

**Comment by annagav** [comment]
@noisecapella I updated the PR. Thank you

**Review by noisecapella — APPROVED** [review]
Thanks, the output JSON looks good now

**Pull request merged**